src/components/ui/common/Actions.js

```diff
@@ -27,6 +27,7 @@ exports.selectDropdownOption = function( opts ) {
         });
     }).then(function(filtered) {
         filtered[0].click();
+        openBtn.click();
```

Comment:
they use this code in the beginning so I guessed its obvious what it does...
any way I will refactor this code..